democracy_club/urls.py

```diff
@@ -150,6 +145,14 @@
         "about/support-us/",
         RedirectView.as_view(url=reverse_lazy("donate")),
     ),
+    path(
+        "about/impact/",
+        RedirectView.as_view(url=reverse_lazy("impact")),
+    ),
```

Comment:
@symroe I've added the `RedirectView` here. Did I misunderstand?

Reply:
Sorry, I was reading too quickly! This is fine :+1: